### PR TITLE
[ci] disable flaky MobileProfiler.Backend test

### PR DESCRIPTION
**Pull request by suo**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #78320

This test is flaky, normally I'd disable using the disable bot but it
doesn't support cpp.

[skip ci]

### CODE REVIEW

**Comment by suo** [comment]
tracking fixing and re-enabling at: https://github.com/pytorch/pytorch/issues/78283

File: test/cpp/lite_interpreter_runtime/test_mobile_profiler.cpp [hunk 1]
```diff
@@ -70,38 +70,38 @@ TEST(MobileProfiler, ModuleHierarchy) {
   ASSERT_TRUE(checkMetaData("aten::add", metadata_name, "top(C)::<unknown>.aten::add", trace_file));
 }
 
-TEST(MobileProfiler, Backend) {
```
Comment (malfet) [inline test/cpp/lite_interpreter_runtime/test_mobile_profiler.cpp:73]: One can skip tests by adding `DISABLED_` prefix to its name

**Review by malfet — APPROVED** [review]
LGTM, but please consider disabling using `DISABLED_` prefix akin to https://github.com/pytorch/pytorch/blob/7ea5fa3dd4c544f2508c5697f103d4ef8f183ed8/test/cpp/tensorexpr/test_loopnest.cpp#L2207

**Pull request closed without merging**